xgriddedaxis/remapper.py

```diff
@@ -1,3 +1,285 @@
+import dask
+import numpy as np
+import sparse
+import xarray as xr
+from scipy.sparse import csr_matrix
+
+_OUTGOING_KEY = 'outgoing'
+_INCOMING_KEY = 'incoming'
+
+
 class Remapper:
-    def __init__(self):
-        ...
+    """
+    An object that facilitates conversion of data between two one dimensional axes.
+    """
+
+    def __init__(
+        self, incoming_axis, outgoing_axis, axis_name, boundary_variable,
+    ):
```

Comment:
I am open to ideas regarding the nomenclature of these parameters.

Reply:
I think these names are fine.